### DISCUSSION TITLE
Unable to run commands, commands being hung up

### DISCUSSION THREAD

**Discussion by micahcb** [community/community#112488] [Actions]
### Select Topic Area

Bug

### Body

I am getting stuck on commands. Some do not even go through some get errors. The first git pull showed this error. The git fsck has been stuck on the first line for 5 minutes and shows no signs of continuing.

(smart) MichaelsMBP157:SmartBetter-website michaelblackburn$ git pull
error: error reading from .git/objects/pack/pack-b3fefdffedc62e5bb30e0d04572a26f6b2b2ebe3.pack: Operation canceled
^C
(smart) MichaelsMBP157:SmartBetter-website michaelblackburn$ git fsck
Checking object directories: 100% (256/256), done.

**Comment by abdalla-am** [comment]
It seems like there might be corruption in your Git repository, leading to errors during operations. Try running git fsck with the --full and --verbose flags for a more thorough check. If the issue persists, consider re-cloning the repository from a remote source.
